src/components/base/JLink/JLink.js

```diff
@@ -0,0 +1,76 @@
+// @flow
+
+import React from 'react'
+import classnames from 'classnames'
+import { omit } from 'lodash-es'
+
+import {
+  checkIsAnchor,
+  checkIsExternal,
+} from 'utils/uri'
+
+import jLinkStyle from './JLink.m.scss'
+import { JLinkInternal } from './JLinkInternal'
+
+export type Theme = 'text-white' | 'text-blue'
+
+// base component with inexact props
+export type JLinkProps = {
```

Comment:
Looks like typing should be `StyledComponent<Theme> & { ... }`?

Reply:
Does not work for me, I need this component to be available completely without styles (meaning – without theme).